src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -18,10 +18,14 @@ import {DownloadResultsState} from "../../../app/store/downloadResults/downloadR
 
 describe(`uploadModal `, () => {
 
+    beforeEach(() => {
+        jest.useFakeTimers();
```

Comment:
This wasn't strictly to fix the tests, but I was getting what seemed like a weird race condition and I didn't think it would hurt to user fake timers in all tests to avoid any knock on effects real timers might have..